src/common/runtime-key.js

```diff
@@ -0,0 +1,12 @@
+define([
+	"../util/string/hash"
+], function( stringHash ) {
+
+return function( fnName, locale, args ) {
+	var hash;
+	args = [].slice.call( args, 0 );
```

Comment:
Actually, this is still needed because of https://github.com/jquery/globalize/blob/2baa3317e2726cc6d5f9c0f8895958f78ab9aa34/src/message-runtime.js#L14. But, I'm adapting this call, so we keep runtimeKey simpler.